### PR TITLE
🐛 Fix pagination for empty chunks

### PR DESCRIPTION
**Pull request by giancarloromeo**
## What do these changes do?
Fix validation issue in case of empty chunks.

See: #6983 

## Related issue/s

- Resolves: #6983 

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by GitHK — APPROVED** [review]
👍

**Review by sanderegg — COMMENTED** [review]
did you actually add a test for checking when the number of pages is 0?

File: packages/pytest-simcore/src/pytest_simcore/examples/models_library.py [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
+        "_meta": {"total": 0, "count": 0, "limit": 4, "offset": 0},
+        "_links": {
+            "self": "https://osparc.io/v2/listing?offset=0&limit=4",
+            "first": "https://osparc.io/v2/listing?offset=0&limit=4",
+            "prev": None,
+            "next": None,
+            "last": "https://osparc.io/v2/listing?offset=0&limit=4",
+        },
+        "data": [],
+    },
```
Comment (giancarloromeo) [inline packages/pytest-simcore/src/pytest_simcore/examples/models_library.py:39]: @sanderegg 
> did you actually add a test for checking when the number of pages is 0?

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks!

**Review by sanderegg — APPROVED** [review]
ok all good I think I missed it previously. thanks a lot

**Pull request merged**